src/Core/src/Platform/iOS/MauiCAKeyFrameAnimation.cs

```diff
@@ -0,0 +1,10 @@
+using CoreAnimation;
+
+namespace Microsoft.Maui.Platform;
+
+public class MauiCAKeyFrameAnimation : CAKeyFrameAnimation
```

Comment:
Should this be internal? I think if that is the case then this is a non-chnaging API really.

Reply:
Oh I guess that could be! Let me change that

Reply:
Done!